pkgs/development/libraries/qt-5/5.15/srcs.nix

```diff
@@ -1,342 +0,0 @@
-# DO NOT EDIT! This file is generated automatically.
```

Comment:
Why are we switching away from this file? I think having one big file is easier to manage then 20 little json files.

Reply:
I'm not sure what it means to "manage" a generated file, but I concatenated them per your request.